.github/actions/setup-android/action.yml

```diff
@@ -23,9 +23,9 @@ runs:
 
   steps:
     - name: Install Flutter
-      uses: subosito/flutter-action@e938fdf56512cc96ef2f93601a5a40bde3801046 # pin@v2.19.0
+      uses: subosito/flutter-action@v2
```

Comment:
Please note that actions other than GitHub's official ones must be pinned to a specific commit.